### DISCUSSION TITLE
grep command

### DISCUSSION THREAD

**Discussion by Zoey2936** [nextcloud/all-in-one#3009] [Questions] [answered]
https://github.com/nextcloud/all-in-one/blob/f86032a7e4ccff2562da39da3ff892dfb1820c98/Containers/mastercontainer/daily-backup.sh#L19
not sure but the grep seems not valid to me, does it not need to be `[0-9]\+` ?

**Comment by szaimen** [comment]
What would bw your suggestion on how to change the grepping?

**Reply by Zoey2936** [reply]
`docker inspect nextcloud-aio-apache --format "{{.HostConfig.PortBindings}}" | grep -o '[0-9]\+' | head -1`

**Reply by Zoey2936** [reply]
@szaimen (sorry for the ping)

**Reply by szaimen** [reply]
All right! If you think this is better then lets do it. Can you provide a PR please?

**Reply by Zoey2936** [reply, marked as answer]
#3171

**Comment by szaimen** [comment]
> does it not need to be [0-9]+

It is already so I am confused by your question...

**Reply by Zoey2936** [reply]
it was wrong formated... fixed now

**Reply by szaimen** [reply]
I think it should work without the backslash. Doesnt it?

**Reply by szaimen** [reply]
Seems to work for me:
![image](https://github.com/nextcloud/all-in-one/assets/42591237/79dbfb49-d934-4337-a848-4d495a88a94a)

**Reply by szaimen** [reply]
This is now released with v7.1.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
